chia/server/ws_connection.py

```diff
@@ -361,37 +358,24 @@ async def time_out(req_id, req_timeout):
                 raise
 
         timeout_task = asyncio.create_task(time_out(message.id, timeout))
-        # TODO: address hint error and remove ignore
-        #       error: Invalid index type "Optional[uint16]" for "Dict[bytes32, Task[Any]]"; expected type "bytes32"
-        #       [index]
-        self.pending_timeouts[message.id] = timeout_task  # type: ignore[index]
+        self.pending_timeouts[message.id] = timeout_task
         await event.wait()
 
-        # TODO: address hint error and remove ignore
-        #       error: No overload variant of "pop" of "MutableMapping" matches argument type "Optional[uint16]"
-        #       [call-overload]
-        #       note: Possible overload variants:
-        #       note:     def pop(self, key: bytes32) -> Event
-        #       note:     def [_T] pop(self, key: bytes32, default: Union[Event, _T] = ...) -> Union[Event, _T]
-        self.pending_requests.pop(message.id)  # type: ignore[call-overload]
+        self.pending_requests.pop(message.id)
         result: Optional[Message] = None
         if message.id in self.request_results:
-            # TODO: address hint error and remove ignore
-            #       error: Invalid index type "Optional[uint16]" for "Dict[bytes32, Message]"; expected type "bytes32"
-            #       [index]
-            result = self.request_results[message.id]  # type: ignore[index]
+            result = self.request_results[message.id]
             assert result is not None
             self.log.debug(f"<- {ProtocolMessageTypes(result.type).name} from: {self.peer_host}:{self.peer_port}")
-            # TODO: address hint error and remove ignore
-            #       error: No overload variant of "pop" of "MutableMapping" matches argument type "Optional[uint16]"
-            #       [call-overload]
-            #       note: Possible overload variants:
-            #       note:     def pop(self, key: bytes32) -> Message
-            #       note:     def [_T] pop(self, key: bytes32, default: Union[Message, _T] = ...) -> Union[Message, _T]
-            self.request_results.pop(result.id)  # type: ignore[call-overload]
+            self.request_results.pop(message.id)
 
         return result
 
+    async def reply_to_request(self, response: Message):
+        if self.closed:
+            return None
+        await self.outgoing_queue.put(response)
+
```

Comment:
```suggestion
```

This was dropped in https://github.com/Chia-Network/chia-blockchain/pull/9327. Looks like re-adding is a result of a rebase conflict?

Reply:
yup